src/styles/workflowPageStyles/index.ts

```diff
@@ -0,0 +1,5 @@
+import { styled } from '@mui/material';
+
+export const Wrapper = styled('div')`
+  margin-top: 2rem;
```

Comment:
Utilizar a contante _spacing_ exportada lá do theme.

```suggestion
  margin-top: ${spacing.S32};
```